manager/src/main/java/io/gingersnapproject/Caches.java

```diff
@@ -70,6 +70,18 @@ public Uni<String> put(String name, String key, String value) {
       return indexUni;
    }
 
+   public Uni<String> putAll(String name, Map<String, String> values) {
```

Comment:
I think we need to return either void or the map of inserted values.

Reply:
It is fine to return Uni<String>. The value is never used, we just avoid an additional chained Uni this way. It isn't the cleanest code, but should be fine.